exchanges/graphcache/src/cacheExchange.ts

```diff
@@ -79,17 +79,26 @@ export const cacheExchange = <C extends Partial<CacheExchangeOpts>>(
     operation: Operation,
     pendingOperations: Operations
   ) => {
+    if (operation.context.reexecuted) {
+      operation.context.reexecuted = false;
```

Comment:
We should remove this line and use it where we transform a cache and network invocation, possibly resulting from requested refetch, to network-only and remove the propety there